packages/react/src/Portal/Portal.tsx

```diff
@@ -43,6 +43,10 @@ function ensureDefaultPortal() {
   }
 }
 
+export const PortalContext = React.createContext<{
```

Comment:
Are there any docs we can provide as a jsdoc here to give a brief description?

```suggestion
/**
 * Provides the ability for component trees to override the portal root container for a sub-set of the experience. 
 * The portal will prioritize the context value unless overridden by their own `containerName` prop, and fallback to the default root if neither are specified 
 */
export const PortalContext = React.createContext<{
```